### ISSUE TITLE
Not working on all apps

### ISSUE THREAD

**Issue by Prayag2** [Luwx/LightlyShaders#13]
This is a great desktop effect but it's not working on all apps. Here's a screenshot:
![image](https://user-images.githubusercontent.com/39525869/126951098-29976330-48ad-4bf2-9d3a-a13f4c8332c5.png)

**Comment by nonetrix** [comment]
Most likely an Electron issue, since it is not using the system tile bars etc

Could be wrong, I am not good at this stuff

**Comment by qawsedrftgzh** [comment]
Yes this is right alt+F3 then other actions -> no border = false
